stable_baselines/td3/td3.py

```diff
@@ -473,3 +456,55 @@ def save(self, save_path, cloudpickle=False):
         params_to_save = self.get_parameters()
 
         self._save_to_file(save_path, data=data, params=params_to_save, cloudpickle=cloudpickle)
+
+
+class Runner(AbstractEnvRunner):
```

Comment:
Why do you need a runner? It seems that you only need to save the `obs` variable.

Reply:
I noticed that other implementations that uses VecEnv use a Runner. I used a Runner here as I feel like that best enables future developers to build on top of it.

Reply:
PPO and A2C use a runner because some additional computation/transformations are needed (computation of the GAE notably) which is not the case of TD3 who only need to fill a replay buffer.
However, at some point, we will need to refactor and unify SAC/DDPG/TD3 which have a lot in common.

Reply:
Ah cool. So perhaps some akin to a "runner" for all three?

Reply:
Not really, more a common method `collect_rollout` that would be part of the `OffPolicy` class, but this is not the subject of this PR.